### ISSUE TITLE
Flip animation for two <li> 's 

### ISSUE THREAD

**Issue by alexeymakarov90** [objectivehtml/FlipClock#187]
How to add flip animation to seconds/minutes ? For example, "01" changes to "02" but the animation goes only to digit "2". How to add hours\minutes\seconds flipping ?

**Comment by objectivehtml** [comment]
I am not really not sure what you mean. There is a clock face for that and each digit flips when required. I am not sure what you are wanting to change.

**Comment by derthiede** [comment]
He wants the whole block of seconds to be animated not only the digit wiche actually changes. I want to have the same feature for my clock. Please let me now if there is already a solution.

**Comment by unr** [comment]
I'm here looking for the same solution.

Currently clock is built like

```
[0][1] : [1][2] : [3][0]
```

With each digit ticking on its own interval. Basically, `[3][0]` becomes `[3][1]`, but the animation only occurs on the `[0]`.

Looking for the ability to group these digits, and animate them together like

```
[01] : [12] : [30]
```

And the entirety of `[30]` flips and animates to `[31]`

**Comment by Leagnus** [comment]
For this сlock to fit small sizes two digits should be grouped together.
But it seems whole algorithm should be rewritten... because one ul contains 2 digit values – before and after.

**Comment by hanmingpan** [comment]
I also have the same issue,I initially thought that can be solved through CSS, but adjacent '<ul>' cannot solve ie8 compatibility.

**Comment by brianespinosa** [comment]
Interesting feature request. To do this would require a complete structure change for the markup of the clock. It's not something you would be able to do with just styles. The way the clock was initially designed was to skeumophically emulate old single digit flip clocks. Any clocks that flip two numbers together are newer and not within the scope of the first version of this project.

I'll talk with @objectivehtml to see what he thinks about implementing something like this in the new version we are working on.

**Comment by avradul** [comment]
1 year later... And so?

**Comment by objectivehtml** [comment]
Long time coming, but I played with this in the new v1 draft release, on the js-es6 branch I just wrote. Currently it doesn't support this, but the methods are much more abstract and it could do this by extending a clock face and playing with the digitizer. I opted out of this by default, but this is no longer coupled so closely to the library. Library also supports alpha characters, negative values, and much more. Going to leave this open so an example can be made at some point for the new version.

**Comment by cf-git** [comment]
Hello everyone ... Has anyone found a solution to this problem?

**Comment by objectivehtml** [comment]
This should work out of the box in the upcoming v1 release, currently in the `js-es6` branch. I have tested this with a couple clock faces and if the value changes, then the animation flips on all the digits that change. This should become master this week. Long time coming on this, but a lot of work went into rewriting the plugin to make features like this possible. Closing this thread. If anyone still has problems, make a new issue for the `js-es6` branch or `master` once it has been merged.
